eng/common/instructions/azsdk-tools/local-sdk-workflow.instructions.md

```diff
@@ -0,0 +1,119 @@
+---
+description: "Guide the user to generate and build SDKs locally for TypeSpec changes"
+---
+
+# Goal
+
+Help the user generate and build SDKs locally from TypeSpec API specifications using the `azure-sdk-mcp` tools.
+
+---
+
+## Part A: Generate SDK Locally
+
+### Step 1: Outline workflow
+
+**Goal**: Ensure the user understands the overall SDK generation and build process before starting.  
+**Actions**:
+
+- Present the high-level steps involved in generating and building SDK locally:
+  1. Select target language
+  2. Verify SDK repository
+  3. Validate repository path
+  4. Identify path to configuration file
+  5. Generate SDK using `azure-sdk-mcp`
```

Comment:
what is `azure-sdk-mcp`? Are you referring the mcp tool name in the mcp.json? Can we avoid that to remove that dependency with the configuration name.

Reply:
Good reminder. I updated it with the name of MCP tools.